index.js

```diff
@@ -140,9 +145,33 @@ const alreadyOnScoreboard = (userId) => {
 };
 
 const plotMemberScore = (member) => {
-  return `${bot.users.cache.get(member.discordId)} (${
-    member.score
-  }) ${":star:".repeat(member.score)}\n`;
+  return `${bot.users.cache.get(member.discordId)} (${member.score
+    }) ${":star:".repeat(member.score)}\n`;
 };
 
+const voiceInteraction = (voiceChannel, audioPath, volume=0.7) => {
+  voiceChannel.join().then(connection => {
+    const dispatcher = connection.play(audioPath, { volume: volume });
+    dispatcher.on("finish", finish => {
+      voiceChannel.leave()
+    })
+  }).catch(err => console.log(err));
+}
```

Comment:
Podia ter usado um `try-catch`, com `async` e  `await ` ai... Algo assim:
```suggestion
const voiceInteraction = async (voiceChannel, audioPath, volume=0.7) => {
  try {
        const connection = await voiceChannel.join();
            const dispatcher = connection.play(audioPath, { volume: volume });
            dispatcher.on("finish", finish => {
              voiceChannel.leave()
            })
  } catch (err) {
      console.log(err)
  }
}
```

Outra coisa, seria legal colocar uma mensagem mais explicativa nesse console.log, porque, caso dê erro em produção, a gente consegue ver pelo console do heroku:

`console.log("Error in join voice channel: ", err);`